sdk/storage/azure-storage-blob/perf.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  - name: packageVersions
+    displayName: PackageVersions (regex of package versions to run)
+    type: string
+    default: '12|source'
+  - name: tests
+    displayName: Tests (regex of tests to run)
+    type: string
+    default: '^(download|upload|list-blobs)$'
+  - name: arguments
+    displayName: Arguments (regex of arguments to run)
+    type: string
+    default: '(10240)|(10485760)|(1073741824)|(5 )|(500 )|(50000 )' 
```

Comment:
I'm wondering if we should think about retooling this to be something similar to the test matrix that @benbp created. This would allow us to do what @kasobol-msft has discussed in the past on allowing multiple perf tests being run at the same time on different VMs against different Storage accounts and make these "magic" regexes clearer and more easily extended to support more options.

Reply:
@mikeharder happy to work with you to get the matrix spun up. I agree it will probably make configuration easier and would be good to use matrix across all perf pipelines. If it's useful, I can generate an initial config to get started with.

Reply:
I agree the perf tooling should be refactored and/or re-designed.  It was created for the scenario of ad-hoc test runs across all languages on a single machine.  As we are moving to one pipeline per language+service, the requirements are changing.

The highest priority right now is to get **some** form of daily perf pipelines running on main to catch regressions, even if the tooling is suboptimal.  It was a conscious choice to get the pipelines running with the current tooling, rather than delaying even longer while the tooling is being re-designed.

Reply:
Another goal is for the perf pipelines to be as similar as possible to the live test pipelines.  So I agree the perf test pipelines should use a matrix similar to the live tests.